### ISSUE TITLE
Using mod as a dependency causes issues with runData in IDE

### ISSUE THREAD

**Issue by Ryorama** [MrCrayfish/MrCrayfishFurnitureMod-Refurbished#69]
I'm working on a mod that uses this mod as a dependency, but when I try using runData in my IDE the instance will crash due to a `ResourceLocationException`. I've tested it with runClient and runServer as well but I don't get the error then which leads me to believe it's only an issue with using runData.
[latest.log](https://github.com/user-attachments/files/17894791/latest.log)

**Comment by MrCrayfish** [comment]
Try removing `'--all'` from data run args

**Comment by MrCrayfish** [comment]
Make sure your datagen run args has `'--mod', your_mod_id`, otherwise you datagen for all mods.

**Comment by Ryorama** [comment]
Yeah, I have that but it still doesn't work. I'll upload my build.gradle to paste bin for you to check. I also tried using the official maven repo as well thinking cursemaven was screwing something up but that didn't fix anything either.
https://pastebin.com/y7QRLfxf

**Comment by MrCrayfish** [comment]
Fixed in 0f0eb49d50c7fb272292d410abe5c8faa782167d
